Project-Aurora/Project-Aurora/Settings/Configuration.cs

```diff
@@ -441,8 +441,14 @@ public class Configuration : Settings
         public SkypeOverlaySettings skype_overlay_settings;
 
         //Debug Settings
-        public bool BitmapTopMost;
-        public bool GSIHttpDebugTopMost;
+        private bool bitmapTopMost;
+        [JsonProperty(PropertyName = "bitmap_debug_topmost")]
```

Comment:
The JsonProperty renaming should only be used for compatibility with the old name of a property in Configuration. It shouldn't be used for new properties, the names used in the json should be the same as the Property unless there's a good reason to change it to something else

Reply:
no longer doing JsonProperty renaming